tests/e2e/daemonset-features/add-sa-collector.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [[ "$(kubectl api-resources)" =~ "operator.openshift.io" ]]; then
```

Comment:
might be better `kubectl api-resources --api-group=operator.openshift.io  -o name` and check if there is any output